frontend/plan/components/ShowMore.tsx

```diff
@@ -1,15 +1,22 @@
 import React, { useState } from "react";
-import PropTypes from "prop-types";
 import Truncate from "react-truncate-markup";
 
+interface ShowMoreProps {
+    more: Element | string;
```

Comment:
`ReactNode` might work here. Also, instead of declaring `children` as an `Element | string`, you can wrap this in `PropsWithChildren<ShowMoreProps>` in the type annotation. A bit cleaner!